### PR TITLE
Updates AutoComps

### PR DESCRIPTION
**Pull request by AMcilraith**
## About the Pull Request

Updates Autocompressor to something more modern

## Why It's Good For The Game

Makes medical more easy and less people die.

## Changelog

🆑
tweak: adds a new autocompressor that has an oxygen tank, to replace the old one
/:cl:

Credits go to @VisVirific for the initial sprites and initial code.

### CODE REVIEW

File: code/code/game/objects/items/weapons/tanks/tanks.dm
Comment (checkraisefold) [inline code/code/game/objects/items/weapons/tanks/tanks.dm:1]: we already have tanks

File: code/code/game/objects/items/devices/auto_cpr.dm [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+		breath_mask_off()
+	QDEL_NULL(breath_mask)
+	QDEL_NULL(tank)
+	return ..()
+
+/obj/item/auto_cpr/update_icon()
+	item_state = "[cpr_mode ? "med_harness_cpr" : "[initial(item_state)]"]"
+
+	if(breath_mask)
+		AddOverlays("mask_[mask_on ? "worn" : "idle"]")
```
Comment (checkraisefold) [inline code/code/game/objects/items/devices/auto_cpr.dm]: all usage of this needs to be fixed to add_overlay

**Review by checkraisefold — CHANGES_REQUESTED** [review]
a la discord comments

File: code/code/game/objects/items/devices/auto_cpr.dm (hunk 1 above)
Comment (checkraisefold) [reply code/code/game/objects/items/devices/auto_cpr.dm]: its still not fixed you did add_overlays instead of add_overlay

Comment (AMcilraith) [reply code/code/game/objects/items/devices/auto_cpr.dm]: Yes I know, life has gotten busy and I am working on fixing other issues I found before amending the commit

**Pull request closed without merging**